pages/api/address/transactions/[address].ts

```diff
@@ -32,22 +33,34 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
       if (req.query.address === '' || req.query.address === undefined) {
         throw new Error(ADDRESS_NOT_PROVIDED_400.message)
       }
+      let transactions = []
       const address = parseAddress(req.query.address as string)
-      const transactions = await fetchAddressTransactions(address)
+
+      // this flag ?server_only=1 tells us to only retrieve what we have in the database
+      const serverOnly = req.query.server_only === '1'
```

Comment:
Small thing, but we have been using camelCase for the request parameters (e.g. in `pages/api/transaction/[transactionId].ts`), so best to user `serverOnly` here to mantain consistency.

Reply:
Or if you think snake_case is better for request parameters, you can argue for it, keep like that, and open an issue to fix it another places

Reply:
done